libmattermost.c

```diff
@@ -2001,11 +2001,16 @@ mm_get_or_create_default_group()
 typedef struct {
 	PurpleRoomlist *roomlist;
 	gchar *team_id;
+	gchar *team_desc;
 } MatterMostTeamRoomlist;
 
 static void
 mm_roomlist_got_list(MattermostAccount *ma, JsonNode *node, gpointer user_data)
 {
+
+#define _MAX_COLS_S "30"
+#define _MAX_COLS_I 33

```

Comment:
Are these intended to be different values?  Can you not also define `_MAX_COLS_S` as `#_MAX_COLS_I` so there's only one variable?